ui/StatusQ/src/StatusQ/Components/StatusMessage.qml

```diff
@@ -73,8 +73,8 @@ Control {
     property StatusMessageDetails messageDetails: StatusMessageDetails {}
     property StatusMessageDetails replyDetails: StatusMessageDetails {}
 
-    property string timestampString: new Date(notification.timestamp).toLocaleTimeString(Qt.locale(), Locale.ShortFormat)
-    property string timestampTooltipString: new Date(notification.timestamp).toLocaleString()
```

Comment:
Just double checking: `notification` is just magically gone?

Reply:
Yeah, this was the error, it was never supposed to be there in StatusMessage :) (some copy and paste error I suppose)

Reply:
I mean, there is no such thing as a `notification` inside StatusQ